extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl

```diff
@@ -73,12 +70,16 @@
       [% " (disabled)" UNLESS target.is_enabled %]
     </td>
   </tr>
+  <tr>
+    <th>&nbsp;</th>
+    <td colspan="2">
+      [% IF target.id == user.id %]
+      <a href="userprefs.cgi?tab=account"><input type = "button" value = "Edit Profile & Preferences"></a>
```

Comment:
nit: remove the spaces between = for html attributes: type="button"
I think the button can just say "Edit Profile". GitHub itself does this same thing --
it has a profile and a settings page, and on the profile there is an Edit Profile button that takes you to the Profile section of settings.